README.md

```diff
@@ -1,13 +1,47 @@
-# alfred-org-capture
+# Ian FitzPatrick's fork of alfred-org-capture
 
-How to get [alfred](https://www.alfredapp.com/) and [org-capture](http://orgmode.org/manual/Capture.html) to play nice.
+This is a fork of
```

Comment:
Point taken ;-) Shall I leave the rest of the readme as is (i.e., in my branch)? Or incorporate the additions & issues into your original text?